gdcdictionary/schemas/original_file.yaml

```diff
@@ -0,0 +1,44 @@
+$schema: "http://json-schema.org/draft-04/schema#"
+
+id: "original_file"
+title: Original File
+type: object
+namespace: https://dcp.bionimbus.org/
+category: original_data
+program: '*'
+project: '*'
+description: "A storage node for the original, raw contents of a study ingested into our system."
```

Comment:
```suggestion
description: "A storage node for the original data files of a study. This node is only populated for dataset level PFBs added to Discovery."
```

Reply:
I think we need to be clear that this node will not have data in the graph. @Avantol13 @jdorsheimer any suggestions on wordsmithing?

Reply:
eh, I don't know if that's absolutely necessary to be in this description. I think it's fine if that's a particular pattern/rule we follow without encoding it here.

Reply:
Sounds good. @AlbertSnows ignore my suggestion.